Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItemDefault.m

```diff
@@ -32,18 +32,27 @@ @interface BPKHorizontalNavigationItemDefault ()
 @property(readonly) UIColor *contentColor;
 @property(nonatomic) CGFloat horizontalSpacing;
 @property(nonatomic) CGFloat verticalSpacing;
+@property(nonatomic) BOOL showNotificationDot;
+@property(nonatomic, strong) UIView *notificationDot;
```

Comment:
This should be `readonly` right?

Reply:
It's in the .m file so 🤷‍♂️

Reply:
It's still helpful though? It expresses the intent better for anyone reading the code even if it has no impact on consumers?

Reply:
Yeah that's fair I'll do the change 👍